### DISCUSSION TITLE
Can I restart the sever?

### DISCUSSION THREAD

**Discussion by timeanddoctor** [Kitware/trame#195] [Q&A]
Can I restart or restart everyday the sever if have a bug?

**Comment by jourdain** [comment]
You can do what ever you want but usually for real deployment you will rely on a launcher to support multi-clients. 
With the [cookiecutter](https://github.com/Kitware/trame-cookiecutter) we provide a Docker setup that bundles everything for you. But if you want to learn more on possible [Cloud setup you can read that documentation guide](https://kitware.github.io/trame/docs/deploy-cloud.html).
